rewrite-java-test/src/test/java/org/openrewrite/java/RenameVariableTest.java

```diff
@@ -245,7 +283,7 @@ void renameForLoopVariables() {
             """
               package org.openrewrite;
 
-              public class A {
+              /** @noinspection ReassignedVariable*/ public class A {
```

Comment:
Didn't expect this comment here; did your IDE add this accidentally?
```suggestion
              public class A {
```